src/engine/uct/node/mod.rs

```diff
@@ -146,21 +146,38 @@ impl Node {
                 .iter()
                 .map(|&m| Node::new(m, self.config))
                 .collect();
+            if self.children.len() <= (game.size() * game.size() / 10) as usize {
+                let player = game.last_move().color().opposite();
+                if game.winner() == player || !self.config.uct.playout_aftermath { //don't pass if we're losing on the board
```

Comment:
Oh, and in theory is would be nice if we could just define `expand()` generically so that we don't care of we supply a `Game` or `Board` as the argument. Right now that's not possible, but it should be possible to change either `Game` or `Board` so that the methods that do the same thing are actually called the same.